packages/proxy/src/providers/google.ts

```diff
@@ -253,9 +284,9 @@ export function googleCompletionToOpenAICompletion(
     object: "chat.completion",
     usage: data.usageMetadata
       ? {
-          prompt_tokens: data.usageMetadata.promptTokenCount,
-          completion_tokens: data.usageMetadata.candidatesTokenCount,
-          total_tokens: data.usageMetadata.totalTokenCount,
+          prompt_tokens: data?.usageMetadata?.promptTokenCount || 0,
```

Comment:
since you're checking `data.usageMetadata` above, why would this change be needed?

Reply:
if I recall it was a typing error that led me here